Documentation/ContentObjects/Fluidtemplate/Index.rst

```diff
@@ -17,6 +17,140 @@ It can be used in :ref:`content elements <t3coreapi:adding-your-own-content-elem
 or to generate content within the top-level object page
 (see :ref:`the example on this page <cobj-fluidtemplate-examples>`).
 
+..  versionadded:: 13.1
+
+    Starting with TYPO3 v 13.1 you can use the content object
+    :ref:`PAGEVIEW <cobj-pageview>` for templates on page level. It reduces
+    the amount of TypoScript needed to render a page in the TYPO3 frontend.
```

Comment:
Maybe also a sentence like:

"The new :typoscript:`PAGEVIEW` content object has very specific conventions and defaults, that requires (and allows) less configuration, with the benefit that following these conventions means less boilerplate code to maintain.
If you follow these conventions, a few directories and files most follow the structure outlined below."

?

Reply:
Added it, please have a look after your vacation.